src/app/app.component.html

```diff
@@ -1,23 +1,2 @@
-<p-button label="PrimeNG"></p-button>
-
-<div class="flex flex-row-reverse flex-wrap">
-  <div
-    class="flex align-items-center justify-content-center w-4rem h-4rem bg-primary font-bold border-round m-2"
-  >
-    1
-  </div>
-  <div
-    class="flex align-items-center justify-content-center w-4rem h-4rem bg-primary font-bold border-round m-2"
-  >
-    2
-  </div>
-  <div
-    class="flex align-items-center justify-content-center w-4rem h-4rem bg-primary font-bold border-round m-2"
-  >
-    3
-  </div>
-</div>
-
-<!-- <span>{{ "KOD_SŁOWNIKA" | translate }}</span> -->
-
-<p>{{ "welcomeMessage" | translate }}</p>
+<app-form></app-form>
+<app-list></app-list>
```

Comment:
usuwamy. W app.component.html powinna jak na razie być tylko linijka 
`<router-outlet></router-outlet>`
bo w tym momencie nie wychwytujesz zmian w routingu i nie wyświetlasz stron per routing

Reply:
Do przeczytania: https://angular.io/tutorial/tour-of-heroes/toh-pt5